test/functional/bumpfee.py

```diff
@@ -34,21 +34,18 @@ def __init__(self):
         super().__init__()
         self.num_nodes = 2
         self.setup_clean_chain = True
+        self.extra_args = [["-prematurewitness", "-walletprematurewitness", "-walletrbf={}".format(i)]
+                           for i in range(self.num_nodes)]
 
-    def setup_network(self, split=False):
-        extra_args = [["-prematurewitness", "-walletprematurewitness", "-walletrbf={}".format(i)]
-                      for i in range(self.num_nodes)]
-        self.nodes = self.start_nodes(self.num_nodes, self.options.tmpdir, extra_args)
-
+    def run_test(self):
         # Encrypt wallet for test_locked_wallet_fails test
         self.nodes[1].node_encrypt_wallet(WALLET_PASSPHRASE)
-        self.nodes[1] = self.start_node(1, self.options.tmpdir, extra_args[1])
+        self.start_node(1)
         self.nodes[1].walletpassphrase(WALLET_PASSPHRASE, WALLET_PASSPHRASE_TIMEOUT)
 
         connect_nodes_bi(self.nodes, 0, 1)
```

Comment:
This is already done by `setup_network`

Reply:
L41 stops node 1 (the `encryptwallet` RPC shuts down the node), so I believe all the p2p connections are lost and we need to reconnect the nodes after restarting node 1.

I've tested removing these two lines and the test fails (the next call to `sync_all()` fails)

Reply:
Sorry, missed that the node shuts down.